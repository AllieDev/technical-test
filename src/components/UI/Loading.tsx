import LoadingButton from "@mui/lab/LoadingButton";

function Loading() {
  return (
    <div className="mapLoading">
      <LoadingButton loading variant="outlined">
        Loading
      </LoadingButton>
    </div>
  );
}

export default Loading;
