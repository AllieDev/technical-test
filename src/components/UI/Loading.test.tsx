import Loading from "./Loading";
import { describe, expect, it } from "vitest";
import { render } from "@testing-library/react";

describe("Loading", () => {
  it("should render", () => {
    const { container } = render(<Loading />);
    expect(container.textContent).toMatchSnapshot();
  });
});
