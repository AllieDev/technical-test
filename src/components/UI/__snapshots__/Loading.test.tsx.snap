// Vitest Snapshot v1

exports[`#App > serial test 1`] = `"Loading"`;

exports[`Loading > should render 1`] = `"Loading.."`;
